### PR TITLE
Some docstrings for actions

### PR DESCRIPTION
**Pull request by darrenburns**
Some docstrings I had on a local branch on this machine.

### CODE REVIEW

File: src/textual/app.py [hunk 1]
```diff
@@ -2375,18 +2375,22 @@ async def action_pop_screen(self) -> None:
...
 
     async def action_back(self) -> None:
+        """Go back to the previous screen (pop the current screen)."""
         try:
             self.pop_screen()
         except ScreenStackError:
             pass
 
     async def action_add_class_(self, selector: str, class_name: str) -> None:
+        """Add a CSS class on the selected widget."""
```
Comment (davep) [inline src/textual/app.py]: Should probably add the `Args` while here.

File: src/textual/app.py [hunk 2]
```diff
@@ -2375,18 +2375,22 @@ async def action_pop_screen(self) -> None:
...
         self.screen.query(selector).add_class(class_name)
 
     async def action_remove_class_(self, selector: str, class_name: str) -> None:
+        """Remove a CSS class on the selected widget."""
```
Comment (davep) [inline src/textual/app.py]: Should probably add the `Args` while here.

File: src/textual/app.py [hunk 3]
```diff
@@ -2375,18 +2375,22 @@ async def action_pop_screen(self) -> None:
...
         self.screen.query(selector).remove_class(class_name)
 
     async def action_toggle_class(self, selector: str, class_name: str) -> None:
+        """Toggle a CSS class on the selected widget."""
```
Comment (davep) [inline src/textual/app.py:2402]: Should probably add the `Args` while here.

**Review by davep — APPROVED** [review]
Suggest adding `Args` where there are arguments, but otherwise good.

File: src/textual/app.py (hunk 3 above)
Comment (darrenburns) [reply src/textual/app.py]: Sorry, I meant to make this PR a draft.

Comment (davep) [reply src/textual/app.py]: But I'm sure this will be a perfect PR and can be pre-approved. :-P

**Pull request merged**